composer.json

```diff
@@ -12,13 +12,13 @@
         "phpunit/phpunit": "^7.5",
         "mockery/mockery": "^1.3",
         "phpstan/phpstan": "^0.12.42",
-        "friendsofphp/php-cs-fixer": "^2.16"
+        "friendsofphp/php-cs-fixer": "^2.16.1"
     },
     "scripts": {
-        "analyze": "vendor/bin/phpstan analyse -l 6 src/",
-        "php-cs-fixer:lint": "vendor/bin/php-cs-fixer fix --verbose --dry-run",
-        "php-cs-fixer:format": "vendor/bin/php-cs-fixer fix",
-        "test:unit": "vendor/bin/phpunit --coverage-text --coverage-clover coverage.xml"
+        "analyze": "phpstan analyse -l 6 src/",
```

Comment:
This makes the assumption that `vendor/bin` is in the `$PATH`. While this is for me, it isn't for everybody.

Reply:
I need to check the priority call (local vs global)

Reply:
I've just test it, it calls the local one instead of the global

So it's OK :+1:

Reply:
No. This is okay for you because you have vendor/bin in your path. Not everybody does. This would need to be reverted before I could consider merging.

Reply:
If I may weigh in…

To the best of my knowledge, composer itself does that. Please see https://getcomposer.org/doc/articles/scripts.md#writing-custom-commands
> Before executing scripts, Composer's bin-dir is temporarily pushed on top of the PATH environment variable so that binaries of dependencies are easily accessible. In this example no matter if the phpunit binary is actually in vendor/bin/phpunit or bin/phpunit it will be found and executed.

Apologies if I side-tracked / misinterpreted things here.

Reply:
Oh that’s cool. Okay, I think I’m happy with this diff :)